invenio_app_ils/documents/jsonresolvers/document_circulation.py

```diff
@@ -82,8 +82,20 @@ def circulation_resolver(document_pid):
             if bucket["key"] in "FOR_REFERENCE_ONLY":
                 items_for_reference_count = bucket["doc_count"]
 
+        active_ill_loans_count = 0
+
+        for hit in loans_result.hits:
```

Comment:
Note: the hits count will not exceed the window size, so it's possible that we will miss some. I don't remember how many results are returned at most (25? 10k?) but it might not ever happen in practice anyway because there are usually only a few items linked to the same document. In case that happens the`available_items_for_loan_count` could be negative, but unlikely.